pkg_resources/extern/__init__.py

```diff
@@ -70,12 +70,21 @@ def install(self):
             sys.meta_path.append(self)
 
 
+# [[[cog
+# import cog
+# from tools.vendored import yield_root_package
+# names = "\n".join(f"    {x!r}," for x in yield_root_package('pkg_resources'))
+# cog.outl(f"names = (\n{names}\n)")
+# ]]]
 names = (
     'packaging',
     'platformdirs',
+    'typing_extensions',
     'jaraco',
     'importlib_resources',
+    'zipp',
```

Comment:
@jaraco, `zipp` was not in this `pkg_resources/extern/__init__.py`, but the `vendored.txt` file says:

> ```
> # required for importlib_resources on older Pythons
> zipp==3.7.0
> ```

Maybe that was a bug? Or is "older Pythons" here referring to Python < 3.7? In that case maybe we can remove it?

Reply:
I'm unsure what older Pythons means, as it's dependent on the version of importlib_resources that's resolved. I would keep a version compatible with whatever importlib_resources declares.